cmake/External/aotriton.cmake

```diff
@@ -5,15 +5,7 @@ if(NOT __AOTRITON_INCLUDED)
   set(__AOTRITON_BUILD_DIR "${CMAKE_CURRENT_BINARY_DIR}/aotriton/build")
   set(__AOTRITON_INSTALL_DIR "${PROJECT_SOURCE_DIR}/torch")
   add_library(__caffe2_aotriton INTERFACE)
-  # Note it is INSTALL"ED"
-  if(DEFINED ENV{AOTRITON_INSTALLED_PREFIX})
-    install(DIRECTORY
-            $ENV{AOTRITON_INSTALLED_PREFIX}/lib
-            $ENV{AOTRITON_INSTALLED_PREFIX}/include
-            DESTINATION ${__AOTRITON_INSTALL_DIR})
-    set(__AOTRITON_INSTALL_DIR "$ENV{AOTRITON_INSTALLED_PREFIX}")
-    message(STATUS "Using Preinstalled AOTriton at ${__AOTRITON_INSTALL_DIR}")
```

Comment:
We need to keep this section so developers can work with AOTriton build from source, but out of PyTorch source.

Reply:
@jeffdaily I'm okay with reinstating this option, but I'd still like to remove the installation steps from the dockerfiles and remove the `install_aotriton.sh`, mainly because I do not want to have to maintain duplicate logic in `aotriton.cmake` and `install_aotriton.sh` wrt parsing the `aotriton_version.txt` file. Do you have any concerns?

Reply:
I'm okay with these suggestions.  I would like for building aotriton from source to become a step for power users only.  They can navigate aotriton version compatibility.  We should make it as easy as possible for the common cases.

Reply:
Reinstated AOTRITON_INSTALLED_PREFIX section